Makefile

```diff
@@ -1098,3 +1098,9 @@ config.status:
 	@echo "should work."
 	@false
 endif
+
+noop:
+	$(MAKE) -C otherlibs noop
+	$(MAKE) -C otherlibs/win32unix noop
+	$(MAKE) -C ocamltest noop
+	$(MAKE) -C runtime noop
```

Comment:
Suppose I'm coming later to this Makefile, it would be hard for me to understand why there is a `noop` rule, and what is the justification for which children Makefile we are dispatching to. (For the first question: the point is to check the dependency generation magic. For the second, I am not sure.) Could there be a comment explaining this?

Reply:
These aren't supposed to be here - sorry - I shall repush the Changes commit.